### PR TITLE
[xToken]add arbitrum aum

### PR DESCRIPTION
**Pull request by noraliucode**
##### Twitter Link:
https://twitter.com/xtokenmarket

##### List of audit links if any:
`xu3lp`: [link](https://github.com/xtokenmarket/xu3lp/tree/master/audit)

##### Website Link:
https://xtoken.market/

##### Current TVL:
$15,135,409

##### Chain:
Ethereum / Arbitrum

##### Coingecko ID (so your TVL can appear on Coingecko): (https://api.coingecko.com/api/v3/coins/list)
xtoken

##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)
8599

##### Short Description (to be shown on DefiLlama):
Passive, Efficient ERC20 Staking Strategies

##### Token address and ticker if any:
Token: `XTK`

Address: `0x7f3edcdd180dbe4819bd98fee8929b5cedb3adeb`

### CODE REVIEW

File: projects/xtoken.js [hunk 1]
```diff
@@ -1,7 +1,8 @@
+const Web3 = require('web3')
```
Comment (0xngmi) [inline projects/xtoken.js:1]: why change this?

**Comment by 0xngmi** [comment]
is this mergeable?

**Comment by noraliucode** [comment]
> is this mergeable?
Hi yes it's mergeable! Thank you!

File: projects/xtoken.js (hunk 1 above)
Comment (noraliucode) [reply projects/xtoken.js:1]: While I am testing, I got this error:
```
Returned values aren't valid, did it run Out of Gas? You might also see this error if you are not using the correct ABI for the contract you are retrieving data from, requesting data from a block number that does not exist, or querying a node which is not fully synced.
``` 
And change config fixed that

**Pull request merged**